.github/workflows/job_build_mlir_linux.yml

```diff
@@ -133,11 +133,7 @@ jobs:
       - name: Install python dependencies
         if: ${{ !steps.cache-restore.outputs.cache-hit }}
         run: |
-          python3 -m pip install -r ${OPENVINO_REPO}/src/bindings/python/wheel/requirements-dev.txt
-          python3 -m pip install -r ${OPENVINO_REPO}/src/frontends/onnx/tests/requirements.txt
-          python3 -m pip install -r ${OPENVINO_REPO}/src/frontends/tensorflow/tests/requirements.txt
-          python3 -m pip install -r ${OPENVINO_REPO}/src/frontends/tensorflow_lite/tests/requirements.txt
-          python3 -m pip install -r ${OPENVINO_REPO}/src/frontends/paddle/tests/requirements.txt
+          python3 -m pip install --require-hashes --allow-unsafe -r ${NPU_COMPILER_REPO}/.github/requirements-dev.txt
```

Comment:
Seems to cause
```
no such option: --allow-unsafe
Error: Process completed with exit code 2.
```
https://github.com/openvinotoolkit/npu_compiler/actions/runs/14457401724/job/40543474597?pr=53

Reply:
yeah, added it to pip install instead of pip compile :)
Fixed in https://github.com/openvinotoolkit/npu_compiler/pull/53/commits/b5179c05efb33f1693e318c6fb8d5743bccc5e8c